src/pages/about-us/AboutUs.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './assets/about-us.module.scss';
+import subpageBackground from '../../assets/subpageBackground.png';
+import firstImage from '../../assets/aboutUsPage/image1.png';
+import secondImage from '../../assets/aboutUsPage/image2.png';
+import IconCircle from '../../icons/IconCircle';
+import AboutUsParagraph from '../../components/about-us-paragraph/AboutUsParagraph';
+import NumberContainerComponent from '../../components/number-container-component/NumberContainerComponent';
+
+export default function AboutUs() {
+    return (
+        <div className={styles.container}>
+            <div>
+                <img src={subpageBackground} alt="subpage background" className={styles.background}/>
+                <div className={styles.firstIconCircle}>
+                    <IconCircle/>
+                </div>
+                <div className={styles.secondIconCircle}>
+                    <IconCircle/>
+                </div>
+            </div>
+            <div className={styles.wrapper}>
+                <div className={styles.header}>
+                    {__('aboutUsPage.header')}
+                </div>
+                <div className={styles.divider}></div>
+                <div className={styles.paragraph}>
```

Comment:
Render paragraphs from array, you can add some boolean to indicate if render image on left/right side